app/importers/reading/google_sheets_importer.rb

```diff
@@ -0,0 +1,68 @@
+# Returns array of CSVs from a Google Drive Folder. Requires a Google Service Account with read permissions for the folder 
+# Service account created as follows:
+# 1. Log into Google Developer Console with the account you wish to associate with this project
+# 2. Create a project
+# 3. Select "Enable APIs and Services"
+# 4. Search for and enable both the Drive API and Sheets API
+# 5. Navigate to IAM & Admin > Service Accounts
+# 6. Create a service account
+# 7. Assign Role Project > Viewer
+# 8. Create Key - this will provide the values used in the environment variables
+# 9. Create a folder on Google Drive to group items for the script to download
+# 10. Share this folder with the service account email (found on the project page or in the key in step 8)
+# 11. Add the folder name to the list of folders in reading_sheets_fetcher.rb
+# 12. Set the following environment variables from the key in step 8: GOOGLE_ACCOUNT_TYPE, GOOGLE_CLIENT_EMAIL, GOOGLE_CLIENT_ID, GOOGLE_PRIVATE_KEY
+
+require 'google/apis/drive_v3'
+require 'google/apis/sheets_v4'
+require 'googleauth'
+require 'fileutils'
+require 'csv'
+
+class GoogleSheetsImporter
+
+  APPLICATION_NAME = 'Student Insights'.freeze
+  SCOPE = [Google::Apis::DriveV3::AUTH_DRIVE_READONLY, Google::Apis::SheetsV4::AUTH_SPREADSHEETS_READONLY]
+
+  def getSheetsFromFolder(folder_name)
+    auth = Google::Auth.get_application_default(scope: SCOPE)
+    sheet_ids = getSheetIds(auth, folder_name)
+    downloadSheetsAsCVS(auth, sheet_ids)
```

Comment:
This is a tiny thing, but for Ruby please use underscore_case like the other Ruby code and keep it all the same style within the file.